modules/ta.js

```diff
@@ -36,12 +36,11 @@ module.exports = class Ta {
                                 return;
                             }
 
-                            let range = moment().subtract(24, 'hours')
+                            //let range = moment().subtract(24, 'hours')
```

Comment:
feel free to directly remove this line ;)

Reply:
OK